app/views/admins/edit.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= link_to("Back to Admin List", admins_path, :class => 'back-link') %>
+<!-- <%= link_to("Back to Admin List", admins_path, :class => 'back-link') %>
```

Comment:
I think this is messing up part of brakeman, the full output is below. Note warnings will also cause brakeman to fail so I recommend just ignoring them, there's an easy way to do it but I forgot, should be easy to Google.

== Errors ==

Error: app/views/user/index.html.erb:32 :: parse error on value "<" (tLT) Could not parse app/views/user/index.html.erb
Location: Could not parse /csce431/cibs-syllabus-organizer/app/views/user/index.html.erb

== Warnings ==

Confidence: Medium
Category: Mass Assignment
Check: PermitAttributes
Message: Potentially dangerous key allowed for mass assignment
Code: params.require(:user).permit(:name, :email, :role)
File: app/controllers/user_controller.rb
Line: 47